client/src/components/Guests.jsx

```diff
@@ -4,14 +4,13 @@ class Guests extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      guestTotal: '1 guest',
-      guestAdult: 1,
-      guestChildren: 0,
-      guestInfant: 0,
       guestColor: '#fff',
       guestText: '#757575',
       guestArrow: 'https://cdn0.iconfinder.com/data/icons/navigation-set-arrows-part-one/32/ExpandMore-512.png',
+      guestPadding: '3px 0 3px 5px',
```

Comment:
I had no clue one could add css styling as state in the component. Thank you Janelle! This will help me out